jobs/build/custom/Jenkinsfile

```
@@ -266,6 +259,21 @@ node {
                 }
             }
 
+            stage('push images to qe quay') {
+                if (majorVersion == "4") {
+                    config_dir = "${env.WORKSPACE}/qe_quay_config"
+                    buildlib.registry_quay_qe_login(config_dir)
+                    base_command = "${doozerOpts} ${include_exclude} --profile ${repo_type} --registry-config-dir=${config_dir}"
+                    command = "${base_command} images:push --to-defaults"
+                    command += " --filter-by-os='.*'"
+                    try {
+                        buildlib.doozer command
+                    } catch (err) {
+                        echo "image push to qe quay failed, but it's okay we can ignore it"
```

we don't want to halt the build or alert anybody, but it's still customary to mark the build unstable when it didn't completely succeed (although i wish we could just mark the stage unstable)

```suggestion
                        currentBuild.description += "\n<br>image push to qe quay did not completely succeed"
                        currentBuild.result = "UNSTABLE"
```